tuf/client/updater.py

```diff
@@ -694,6 +713,22 @@ def __init__(self, repository_name, repository_mirrors):
     # Save the validated arguments.
     self.repository_name = repository_name
     self.mirrors = repository_mirrors
+    self.targets_map_file = None
+
+    # If targets_map_filename is provided, load the targets_map_file.
+    if targets_map_filename is not None:
+      securesystemslib.formats.PATH_SCHEMA.check_match(targets_map_filename)
+
+      try:
+        self.targets_map_file = securesystemslib.util.load_json_file(\
```

Comment:
No need to escape the newline, there is implied line continuation inside parentheses.